internal/pkg/encoding/json/schema/generate.go

```diff
@@ -109,9 +109,7 @@ func getDefaultValueFor(schema *jsonschema.Schema, level int) interface{} {
 			return `3e4666bf-d5e5-4aa7-b8ce-cefe41c7568a`
 		}
 		return ``
-	case `number`:
-		fallthrough
-	case `integer`:
+	case `number`, `integer`:
```

Comment:
Narazil som na to, ze niektore `switch` stms su zapisane zbytocne zlozito, tak som to rovno fixol.